composer.json

```diff
@@ -20,7 +20,7 @@
   ],
   "require": {
     "php": ">=5.6",
-    "symfony/console": "~2.8|3.*|4.*",
+    "symfony/console": "~2.8|3.*|4.*|5.*",
```

Comment:
Can't we just use ">=2.8" at this point?

Reply:
I think your right. With a new release it should drop non supported versions and update the minimum php version supported to 7 or above.